packages/shared/src/hooks/companion/useBackgroundRequest.ts

```diff
@@ -14,7 +14,7 @@ export const useBackgroundRequest = (
   const client = useQueryClient();
 
   useRawBackgroundRequest(({ key, ...args }) => {
-    if (!enabled || !isQueryKeySame(key, queryKey)) {
+    if (!enabled || !isQueryKeySame([key], queryKey)) {
```

Comment:
Could this already be array in some cases, looking at `CompanionEngagements` and usage it has the `Array.isArray` check. Could you check to make sure it does not break?

Reply:
Good point yes!